classes/Meta.php

```diff
@@ -330,6 +330,7 @@ public static function getHomeMetas($idLang, $pageName)
         $ret['meta_title'] = (isset($metas['title']) && $metas['title']) ? $metas['title'] : Configuration::get('PS_SHOP_NAME');
         $ret['meta_description'] = (isset($metas['description']) && $metas['description']) ? $metas['description'] : '';
         $ret['meta_keywords'] = (isset($metas['keywords']) && $metas['keywords']) ? $metas['keywords'] : '';
+        $ret = Meta::completeMetaTags($ret, $ret['meta_title']);
```

Comment:
What is the point of replacing a value in the array with the same value ?

Reply:
If page number is > 1, it will be appended to `$ret['meta_title']` on controllers like `new-products`, `best-stales`, `prices-drop`

NOTE: 2nd argument is not used in this case, so the following code will have same result: `$ret = Meta::completeMetaTags($ret, '');`